<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">QFrame{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0.591, stop:0.227273 rgba(0, 47, 95, 255), stop:1 rgba(0, 0, 45, 255));
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>60</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:none;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QFrame" name="frame_bttn">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>50</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: none;</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QPushButton" name="btn_quit">
               <property name="minimumSize">
                <size>
                 <width>15</width>
                 <height>15</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>17</width>
                 <height>17</height>
                </size>
               </property>
               <property name="toolTip">
                <string>Close</string>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton {
	border: none;
	border-radius: 8px;
	background-color: rgb(255, 0, 0);
}
QPushButton:hover {
	background-color: rgba(255, 0, 0, 150);
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_maximize">
               <property name="minimumSize">
                <size>
                 <width>15</width>
                 <height>15</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>17</width>
                 <height>17</height>
                </size>
               </property>
               <property name="toolTip">
                <string>Maximize</string>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton {
	border: none;
	border-radius: 8px;
	background-color: rgb(85, 255, 127);
}
QPushButton:hover {
	background-color: rgba(85, 255, 127, 150);
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_minimize">
               <property name="minimumSize">
                <size>
                 <width>15</width>
                 <height>15</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>17</width>
                 <height>17</height>
                </size>
               </property>
               <property name="toolTip">
                <string>Minimize</string>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton {
	border: none;
	border-radius: 8px;
	background-color: rgb(255, 170, 0);
}
QPushButton:hover {
	background-color: rgba(255, 170, 0, 150);
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="styleSheet">
          <string notr="true">background-color: none;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QFrame" name="Login_Frame">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>149</y>
            <width>391</width>
            <height>241</height>
           </rect>
          </property>
          <property name="maximumSize">
           <size>
            <width>430</width>
            <height>250</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QFrame { 
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0.0227273 rgba(150, 0, 183, 150), stop:1 rgba(255, 255, 255, 150));
border-radius: 8px;

}</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QLabel" name="label">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>381</width>
             <height>41</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(225, 255, 255);
background-color: none;
text-align: center;
</string>
           </property>
           <property name="text">
            <string>&lt;STRONG&gt;ĐĂNG NHẬP TÀI KHOẢN&lt;/STRONG&gt;</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEdit">
           <property name="geometry">
            <rect>
             <x>70</x>
             <y>90</y>
             <width>261</width>
             <height>21</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit {
background-color: rgba(188, 183, 255, 160);
border-radius: 10px;
border-color: rgb(0, 255, 255);
color: rgb(255,255,255);
}</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEdit_3">
           <property name="geometry">
            <rect>
             <x>70</x>
             <y>140</y>
             <width>261</width>
             <height>21</height>
            </rect>
           </property>
           <property name="toolTip">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit {
background-color: rgba(188, 183, 255, 160);
border-radius: 10px;
border-color: rgb(0, 255, 255);
color: rgb(255,255,255);
}</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
          <widget class="QPushButton" name="LoginButton">
           <property name="geometry">
            <rect>
             <x>160</x>
             <y>180</y>
             <width>81</width>
             <height>21</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
background-color: rgb(255, 85, 255,160);
border-radius: 8px;
color: rgb(255, 255, 255);
font: bold;
}
QPushButton:hover {
background-color: rbg(249, 219, 255);
}</string>
           </property>
           <property name="text">
            <string>Đăng nhập</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QLabel" name="label_3">
           <property name="geometry">
            <rect>
             <x>70</x>
             <y>120</y>
             <width>61</width>
             <height>16</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: none;
color: rgb(255,255,255);
</string>
           </property>
           <property name="text">
            <string>Mật khẩu:</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_4">
           <property name="geometry">
            <rect>
             <x>70</x>
             <y>60</y>
             <width>101</width>
             <height>31</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>8</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: none;
color: rgb(255,255,255);
</string>
           </property>
           <property name="text">
            <string>Tên tài khoản:</string>
           </property>
          </widget>
          <widget class="QCheckBox" name="checkBox">
           <property name="geometry">
            <rect>
             <x>260</x>
             <y>170</y>
             <width>111</width>
             <height>21</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">font: bold;
text: &quot;Lưu đăng nhập?&quot;;</string>
           </property>
           <property name="text">
            <string>Lưu đăng nhập?</string>
           </property>
          </widget>
          <widget class="QPushButton" name="ConvertButton">
           <property name="geometry">
            <rect>
             <x>80</x>
             <y>210</y>
             <width>251</width>
             <height>21</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <weight>75</weight>
             <italic>false</italic>
             <bold>true</bold>
             <underline>false</underline>
             <strikeout>false</strikeout>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
background-color: rgb(255, 85, 255,0);
border-radius: 8px;
color: rgb(170, 170, 255);
}
QPushButton:hover {
background-color: none;
color: rgb(255,255,255);
}</string>
           </property>
           <property name="text">
            <string>Không có tài khoản? Đăng ký ngay</string>
           </property>
           <property name="autoRepeat">
            <bool>true</bool>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
